package.json

```diff
@@ -16,17 +16,17 @@
 	},
 	"devDependencies": {
 		"@rollup/plugin-node-resolve": "^10.0.0",
-		"autoprefixer": "^9.8.6",
-		"eslint": "^7.13.0",
-		"postcss": "^7.0.35",
-		"postcss-at-rules-variables": "^0.1.10",
-		"postcss-mixins": "^6.2.3",
-		"postcss-nested": "^4.2.3",
+		"autoprefixer": "^10.0.4",
+		"eslint": "^7.14.0",
+		"postcss": "^8.1.10",
+		"postcss-at-rules-variables": "^0.2.0",
```

Comment:
There should be `0.2.1`: https://github.com/Scrum/postcss-at-rules-variables/issues/205

Reply:
Done.